tests/vmi_lifecycle_test.go

```diff
@@ -323,12 +323,13 @@ var _ = Describe("[rfe_id:273][crit:high][vendor:cnv-qe@redhat.com][level:compon
 					List(metav1.ListOptions{LabelSelector: "kubevirt.io=virt-launcher"})
 				Expect(err).To(BeNil(), "Should list virt-launchers")
 				var launcher k8sv1.Pod
```

Comment:
this var isn't used now, right?

Reply:
actually it is. it's the return value of the function if it's found.

Reply:
Good catch. thank you.